webpack.config.js

```diff
@@ -249,40 +249,7 @@ module.exports = async (env) => [
         },
         {
           test: /\.(jpg|jpeg|png|gif)$/i,
-          include: [
-            {
-              or: [
-                (input) => posixPath(input).includes("src"),
-                (input) => posixPath(input).includes("dist/lib/assets/images"),
-                (input) => posixPath(input).includes("node_modules/@kaoto/kaoto-ui/dist/lib/assets/images"),
-                (input) => posixPath(input).includes("node_modules/@patternfly/patternfly/assets/images"),
-                (input) => posixPath(input).includes("node_modules/@patternfly/react-styles/css/assets/images"),
-                (input) => posixPath(input).includes("node_modules/@patternfly/react-core/dist/styles/assets/images"),
-                (input) =>
-                  posixPath(input).includes(
-                    "node_modules/@patternfly/react-core/node_modules/@patternfly/react-styles/css/assets/images"
-                  ),
-                (input) =>
-                  posixPath(input).includes(
-                    "node_modules/@patternfly/react-table/node_modules/@patternfly/react-styles/css/assets/images"
-                  ),
-                (input) =>
-                  posixPath(input).includes(
-                    "node_modules/@patternfly/react-inline-edit-extension/node_modules/@patternfly/react-styles/css/assets/images"
-                  ),
-              ],
-            },
-          ],
-          use: [
-            {
-              loader: "url-loader",
```

Comment:
Shouldn't we remove the url-loader from devDependencies then?

Reply:
Yes, you're right, I got confused about whether `url-loader` was already part of `Webpack`. I removed the deprecated loaders from the `devDependencies` array and adjusted the configuration to use `asset`.